plugins.txt

```diff
@@ -28,13 +28,13 @@ copyartifact:722.v0662a_9b_e22a_c
 coverage:1.13.0
 credentials:1337.v60b_d7b_c7b_c9f
 credentials-binding:657.v2b_19db_7d6e6d
-customizable-header:83.vf51a_828deb_b_e
+customizable-header:96.v295683015391
```

Comment:
- https://plugins.jenkins.io/customizable-header/releases/#version_88.v1a_dfcde5e689
  - We do not have any [`systemMessage` directive in JCasC for release.ci](https://github.com/jenkins-infra/kubernetes-management/blob/main/config/jenkins_release.ci.jenkins.io.yaml) so https://github.com/jenkinsci/customizable-header-plugin/pull/91 is not breaking for us
- https://plugins.jenkins.io/customizable-header/releases/#version_92.v0418e15d00b_c
  - https://github.com/jenkinsci/customizable-header-plugin/pull/92 looks good 
- https://plugins.jenkins.io/customizable-header/releases/#version_94.v8515ce3572f4
- https://plugins.jenkins.io/customizable-header/releases/#version_96.v295683015391


LGTM